README.md

```diff
@@ -93,6 +93,22 @@ In the same fashion, this auto discovery can be disabled with `--no-discovery`.
 If not further specified, `ros2 bag record` will create a new folder named to the current time stamp and stores all data within this folder.
 A user defined name can be given with `-o, --output`.
 
+#### Splitting recorded bag files
+
+rosbag2 offers the capability to split bag files when they reach a maximum size, denoted as the batch size. By default rosbag2 will record all data into a single bag file, but this can be changed using the CLI options.
+
+For example, `ros2 bag record -a -b 100000` will split the bag files when they become greater than 100 kilobytes.
+
+Note: the batch size's units are in bytes and must be greater than `86016`. This option defaults to `0`, which means data is written to a single file.
+
+#### Recording with compression
+
+By default rosbag2 does not record with compression enabled. However, compression can be specified using the following CLI options.
+
+For example, `ros2 bag record -a --compression-mode file --compression-format zstd` will record all topics and compress each file using the [zstd](https://github.com/facebook/zstd) compressor.
+
+Currently, the only `compression-mode` available is `file` and only `compression-format` available is `zstd`. Both of these options default to `none`. To use a compression format, a compression mode must be specified. It is recommended to use this feature with the batch size option.
```

Comment:
Message compression was added in in https://github.com/ros2/rosbag2/pull/418

Reply:
I'll take the opportunity to update the CLI help message.